src/main/webapp/script.js

```diff
@@ -18,3 +18,48 @@ function createElementByTag(text, tag) {
   element.innerText = text;
   return element;
 }
+
+/**
+ * parse.js
+ *
+ * MediaWiki API Demos
+ * Demo of `Parse` module: Parse content of a page
+ *
+ * MIT License
+ */
+
+async function parseData() {
+    // Parse data from single page.
+    const url = "https://en.wikibooks.org/w/api.php?" +
+    new URLSearchParams({
+        origin: "*",
+        action: "parse",
+        page: "Cookbook:Racuszki",
+        format: "json",
+        prop: "text",
+    });
+
+    const request = await fetch(url);
+    const json = await request.json();
```

Comment:
I'd suggest not using `json` as a variable name. `JSON` is a global object in JS. Calling a variable `json` is a bit confusing.

Perhaps `body`?